services/vcpkg/vcpkg-version-helpers.js

```diff
@@ -0,0 +1,17 @@
+import { InvalidResponse } from '../index.js'
+
+export function parseVersionFromVcpkgManifest(manifest) {
+  if (manifest['version-date']) {
```

Comment:
**thought**: It would have been better to also validate the schema directly inside the helper. However, this would require working around the current validation API in place through the service class and directly using [`validate`](https://github.com/njakob/shields/blob/23c0406bedfc6930735e8f5ea75dfe34faf1f290/core/base-service/validate.js#L5) or the Joi schema validation.

However, looking at other usages of `fetchJsonFromRepo`, I left the schema validation outside of this parsing helper.